app/scripts/lib/segment.js

```diff
@@ -0,0 +1,101 @@
+import Analytics from 'analytics-node'
+
+const isDevOrTestEnvironment = Boolean(
+  process.env.METAMASK_DEBUG || process.env.IN_TEST,
+)
+const { SEGMENT_WRITE_KEY } = process.env
```

Comment:
I used nullish coalescing to prevent eslint from automatically destructuring these. You are correct that that doesn't work. I also tried just using `process.env.SEGMENT_WRITE_KEY` inline but this made the ternaries later in the file unwieldy.

Reply:
Huh, interesting solution! I'll have to remember that.